ee/candi/cloud-providers/vsphere/terraform-modules/master-node/main.tf

```diff
@@ -201,4 +201,5 @@ resource "vsphere_virtual_machine" "master" {
       firmware,
     ]
   }
+  wait_for_guest_net_routable = var.wait_for_guest_net_routable
```

Comment:
Why to not set parameter to false directly ? Why we need variable ?

Reply:
I assume that someone might want to override this behavior, so to avoid redoing it twice, I moved it to variables.

Reply:
How user can set this variable to true?

Reply:
Right now, this option is not available. I'm not sure if we need to complicate this now, but on the other hand, I'm afraid that during testing, we will find out that we need to add the configuration for this option.

Reply:
ok, fixed

Reply:
Approved as long as it is not a breaking change (needs to be tested).